src/config/dynamic-theme-fixes.config

```diff
@@ -2347,6 +2347,118 @@ CSS
 
 ================================
 
+tweakers.net
+
+CSS
+.ankeiler>a.commentCount {
+    background-image: url(../../../g/icons/commentcounter_small_bg.svg) !important;
+    text-decoration: none !important;
+}
+.thumb.category {
+   border: none !important;
+}
+#categoryBrowser li a{
+     background: none !important;
+}
+
+#categoryBrowser li.more {
+    background-image: url("../../../../g/if/categories/arrows.png") !important;
+    background-color: 
+    background-position: 252px 10px !important;
+    background-repeat: no-repeat !important;
+}
+
+#categoryBrowser.sidebar li.more {
+    background-position: 282px 10px;
+}
+
+a.showMoreItems {
+   background-image: none !important;
+}
+
+.corner::before,.corner::after {
+    border: 6px solid #333 !important;
```

Comment:
Please, check how the fix looks in light mode. Use `${#333}` pattern putting a source color into `${}`, later the changed value will be substituted.

Reply:
It looks like this ![https://i.imgur.com/jt8OGPJ.png] ${#333}
And like this ![https://i.imgur.com/V2FMjyx.png] with #333 So I don't think it's handy to use ${}
It looks like this in light mode ![https://i.imgur.com/T8aDoYp.png]